app/src/App.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * ---------------------------------------------------------------------
+ * Copyright (c) 2020 EclipseSource Munich
+ * Licensed under MIT
+ * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
+ * ---------------------------------------------------------------------
+ */
+import JsonFormsEditor, {
+  defaultSchemaDecorators,
+  propertySchemaProvider,
+} from '@jsonforms/editor';
+import React from 'react';
+
+export const App = () => (
+  <JsonFormsEditor
+    schemaProviders={[propertySchemaProvider]}
+    schemaDecorators={defaultSchemaDecorators}
```

Comment:
would it make sense to define the two `default` arrays in the app, rather than in the library ?

Reply:
I would like to being able to run the editor without any customization/props but we should handle this in a different ticket I think.